build_library/catalyst.sh

```diff
@@ -62,8 +62,8 @@ storedir="$CATALYST_ROOT"
 distdir="$DISTDIR"
 envscript="$TEMPDIR/catalystrc"
 port_logdir="$CATALYST_ROOT/log"
-repo_basedir="/mnt/host/source/src/third_party"
```

Comment:
was this value wrong before or it is just a mapping? it should have been /mnt/host/source/src/scripts/third_party?

Reply:
It wasn't wrong before, the `third_party` directory was and still is mounted separately there. It just doesn't contain these repositories anymore.

Yeah, maybe a variable wouldn't be a bad idea, although there are several references to the location in other git repos too.